### PR TITLE
Frequently run build to avoid stale build info

### PR DESCRIPTION
**Pull request by sherifnada**
## What
Run the build once a day to make sure we always have up-to-date build information.

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: Airbyte CI
 
-on: push
+on:
+  schedule:
+    - cron: '0 13 * * *'
```
Comment (sherifnada) [inline .github/workflows/gradle.yml:5]: 1pm UTC

Comment (michel-tricot) [reply .github/workflows/gradle.yml:5]: Why not every hour?

**Review by michel-tricot — APPROVED** [review]

**Comment by jrhizor** [comment]
good idea

**Review by cgardens — APPROVED** [review]

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (sherifnada) [reply .github/workflows/gradle.yml:5]: We could do something a little more frequent like 6 hours, but 1 hour will flood the version control channel with notifications in a way that makes it hard to see what changes are going through. Changed it to 6 hours.

Comment (cgardens) [reply .github/workflows/gradle.yml]: lol. i think our build is too slow to do once an hour. if you want to wait until green to do something, it only means 50% of every hour will you actually be able to take action.

**Pull request merged**